qcelemental/models/molecule.py

```diff
@@ -186,25 +194,27 @@ class Molecule(ProtoModel):
         )
     fragments: Optional[List[Array[np.int32]]] = Schema(
         None,
-        description="An indication of which sets of atoms are fragments within the Molecule. This is a list "
-                    "of 1-D array-like objects. Each entry in the list indicates a new fragment. The index "
-                    "of this list matches the index of ``fragment_charges`` and ``fragment_multiplicities``. "
+        description="An indication of which sets of atoms are fragments within the Molecule. This is a list of shape "
+                    "(NFrag) of 1-D array-like objects of arbitrary shape. Each entry in the list indicates a new "
```

Comment:
* arbitrary length, not shape
* better explicitly say "0-indexed" atoms whenever either could be true.